docs/sources/mimir/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -44,7 +44,7 @@ For these reasons, run the ingesters on disks such as SSDs that have fast disk s
 
 ### Ensure caching is enabled
 
-The querier supports caching to reduce the number API calls to the long-term storage.
+The Querier supports caching to reduce the number API requests to the long-term storage.
```

Comment:
```suggestion
The querier supports caching to reduce the number API requests to the long-term storage.
```
We are not capitalizing all of the components. And ingester, for example, is just the ingester. Please revert that bit.

Reply:
Not sure why my comment is duplicated...